dotnetapp-buildcontainer/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM microsoft/dotnet:core
```

Comment:
>microsoft/dotnet:core [](start = 5, length = 21)

I think our samples should be using an explicit version as showcasing the best practices and avoiding the risk of breaking.